src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts

```diff
@@ -45,6 +45,7 @@ export class SearchableDropdownComponent extends BaseComponent implements Contro
     @Input() rows: number;
     @Input() totalRecords: number;
     @Input() placeholder = '';
+    @Input() persistentPlaceholder: string;
```

Comment:
persistentPlaceholder should be a boolean, if this is true then placeholder is always showed.

Template:

[label]="persistentPlaceholder ? placeholder : valueString"